frontend/src/lib/utils.ts

```diff
@@ -304,3 +304,14 @@ export function generateUID(): string {
 export function getEmbeddingIdClassName(embeddingId: string): string {
   return `stAppEmbeddingId-${embeddingId}`
 }
+
+/**
+ * Returns embed URL of this Streamlit app
+ */
+export function getEmbedURL(): string {
+  let windowToGetURLFrom: Window = window
+  if (isInChildFrame()) {
+    windowToGetURLFrom = window.parent
+  }
+  return `${windowToGetURLFrom.location.href.split("?")[0]}?embed=true`
```

Comment:
We are in the process of deprecating `embedded` param in favour of `embed`. So we know already that URL params can change names of become obsolete. That's why I'd extract `embed` as a constant, maybe even kept in a separate file and import it whenever necessary.